### ISSUE TITLE
WindowsError: types.yaml - No such file or directory

### ISSUE THREAD

**Issue by big-husky** [log2timeline/plaso#2180]
**Description of problem:**

log2timeline.exe fails to run on windows with a traceback.

**Command line and arguments:**

 .\log2timeline.exe -z EST5EDT --logfile ..\test\e\plaso-error.log ..\test\e\storage.plaso 'D:\cases\EVIDENCE.E01'

**Source data:**

Crashed same way on two different images. First was a NTFS volume, second was a disk from an Android.

**Plaso version:**

20180930

**Operating system Plaso is running on:**

Windows 10 1803

**Installation method:**

zip package & Visual C++ 2008 redistributable

** IMPORTANT ** - I removed 'hadoop.yaml' from the artifacts folder so that I could get past that startup issue.  This has been reported in issue #2170.  

**Debug output/tracebacks:**

NOTE:  The folder 'esedb_plugins' does not exist in the 'parsers' folder.

Source type     : storage media image
Processing time : 00:00:00

Processing started.
Process Worker_00:
Traceback (most recent call last):
  File "multiprocessing\process.py", line 258, in _bootstrap
  File "plaso\multi_processing\base_process.py", line 307, in run
  File "plaso\multi_processing\worker_process.py", line 143, in _Main
  File "plaso\engine\worker.py", line 97, in __init__
  File "plaso\engine\extractors.py", line 62, in __init__
  File "plaso\engine\extractors.py", line 134, in _InitializeParserObjects
  File "plaso\parsers\manager.py", line 337, in GetParserObjects
  File "plaso\parsers\interface.py", line 86, in __init__
  File "plaso\parsers\interface.py", line 129, in EnablePlugins
  File "plaso\parsers\esedb_plugins\interface.py", line 61, in __init__
  File "plaso\parsers\esedb_plugins\interface.py", line 302, in _ReadDefinitionFile
IOError: [Errno 2] No such file or directory: u'C:\\plaso\\PLASO-~2\\plaso\\parsers\\esedb_plugins\\types.yaml'

**Comment by soundwave01** [comment]
I'm getting the same error on 20180930 but with a RHEL7 pip install

```
Traceback (most recent call last):
  File "/opt/rh/python27/root/usr/lib64/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/multi_processing/base_process.py", line 307, in run
    self._Main()
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/multi_processing/worker_process.py", line 143, in _Main
    self._processing_configuration.parser_filter_expression))
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/engine/worker.py", line 97, in __init__
    parser_filter_expression=parser_filter_expression)
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/engine/extractors.py", line 62, in __init__
    parser_filter_expression=parser_filter_expression)
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/engine/extractors.py", line 134, in _InitializeParserObjects
    parser_filter_expression=parser_filter_expression)
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/parsers/manager.py", line 337, in GetParserObjects
    parser_object = parser_class()
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/parsers/interface.py", line 86, in __init__
    self.EnablePlugins([])
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/parsers/interface.py", line 129, in EnablePlugins
    plugin_object = plugin_class()
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/parsers/esedb_plugins/interface.py", line 61, in __init__
    self._fabric = self._ReadDefinitionFile(self._DEFINITION_FILE)
  File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/parsers/esedb_plugins/interface.py", line 302, in _ReadDefinitionFile
    with open(path, 'rb') as file_object:
IOError: [Errno 2] No such file or directory: u'/opt/rh/python27/root/usr/lib/python2.7/site-packages/plaso/parsers/esedb_plugins/types.yaml'
```

**Comment by ggala** [comment]
Copy the contents of 'https://github.com/log2timeline/plaso/tree/master/plaso/parsers' into 'C:\plaso\parsers\' to get it to 'work'.

**Comment by Onager** [comment]
This is due some missing yaml files in the original Windows release of 20180930 - I've just uploaded a new version of the Windows package to https://github.com/log2timeline/plaso/releases/tag/20180930 so please try again.

I've added a fix for this to https://github.com/log2timeline/plaso/pull/2191

**Comment by big-husky** [comment]
I've tested this new windows release of 20180930 and can confirm it fixes this issue and #2170 

Thank you!
